### DISCUSSION TITLE
How was the new human TF list defined?

### DISCUSSION THREAD

**Discussion by lucygarner** [aertslab/SCENIC#376] [* General SCENIC questions] [answered]
Hi @cbravo93 @SeppeDeWinter, 

I was wondering how the new human TF list (https://resources.aertslab.org/cistarget/tf_lists/allTFs_hg38.txt) was defined. I was previously using the list of 1639 human TFs from Lambert et al., Cell, 2018. 

In your SCENIC+ paper, you say: "Genes were considered as TFs if they were included in the TF list available on http://humantfs.ccbr.utoronto.ca/". However, the `allTFs_hg38.txt` file contains 1892 TFs, so I was wondering what the extra TFs are?

Best wishes,
Lucy

**Comment by cbravo93** [comment, marked as answer]
Hi @lucygarner !

The TF list was made based on the TF annotations from all motif collections. It is possible there are a few that are annotated to a motif, but are not TFs. The intersection with humantfs was only done in the pbmc and melanoma analyses.

Hope this helps!

C

**Comment by lucygarner** [comment]
Ok thank you, very helpful. So I should be fine to carry on using the list of 1639 human TFs that I was previously using?

Could you clarify what you mean by "The intersection with humantfs was only done in the pbmc and melanoma analyses". Did you use "allTFs_hg38.txt" for gene regulatory network inference?

**Comment by cbravo93** [comment]
I'd recommend updating to use: https://github.com/aertslab/scenicplus/blob/main/resources/allTFs_hg38.txt (with 1,8K TFs). The list overlaps a lot with the previous, but you may miss some TFs for which we actually have a motif now in the SCENIC+ motif collection.

This list was intersected with the TFs in humantfs for the melanoma and pbmc analyses, but i'd recommend to use the full list

Cheers!

C

**Reply by lucygarner** [reply]
Ok thank you. Do you recommend this even though some of the TFs in https://github.com/aertslab/scenicplus/blob/main/resources/allTFs_hg38.txt (165; e.g. ID1, ID2, UBTF, EZH2) are not classified as TFs according to Lambert et al., Cell, 2018? I am trying to understand what is most important about the list of TFs supplied.

**Comment by cbravo93** [comment]
Yes. We checked the difference between the two lists, and quite some of the missing ones are true TFs. You can always choose to filter them a posteriori.

Cheers!

C
